tests/jest/setup.ts

```diff
@@ -0,0 +1,4 @@
+import '@testing-library/jest-dom';
```

Comment:
importing jest assertion extensions from @testing-library at the root level: 
a) because we can't call jest-setup from individual packages yet
b) this should probably be a global inclusion, I imagine we'll want to use this for most client side packages.